vuu-ui/packages/vuu-layout/src/layout-persistence/RemoteLayoutPersistenceManager.ts

```diff
@@ -0,0 +1,137 @@
+import { LayoutMetadata } from "@finos/vuu-shell";
+import { LayoutPersistenceManager } from "./LayoutPersistenceManager";
+import { LayoutJSON } from "../layout-reducer";
+import { defaultLayout } from "./data";
+
+const baseURL = "http://127.0.0.1:8081/api";
+const metadataSaveLocation = "layouts/metadata";
+const layoutsSaveLocation = "layouts";
+
+export class RemoteLayoutPersistenceManager
+  implements LayoutPersistenceManager
+{
+  createLayout(
+    metadata: Omit<LayoutMetadata, "id" | "created">,
+    layout: LayoutJSON
+  ): Promise<LayoutMetadata> {
+    return new Promise((resolve, reject) =>
+      fetch(`${baseURL}/${layoutsSaveLocation}`, {
+        headers: {
+          "Content-Type": "application/json",
+        },
+        method: "POST",
```

Comment:
I think I'm a bit of a strongly typed fiend, too used to Java, because I want to see this (and the others) as like `HttpRequest.POST`. I suppose this isn't a thing we can get out of the box with Typescript?

Reply:
unfortunately the type definition that comes with 'node-fetch' package has type string for the request "method". It would have been nice if it was "GET" | "POST" | "PUT" and so on

Reply:
we could implement our own fetch() and enforce more strict type for method if we wanted

Reply:
I'm not concerned enough by it to force a refactor, seems like unnecessary additional code for a very small gain. Feel free to implement it if you want to.

Reply:
discussed offline. no change needed